css/_base.scss

```diff
@@ -31,8 +31,8 @@ body {
     font-size: 12px;
     font-weight: 400;
     overflow: hidden;
-    color: $defaultColor;
-    background: $defaultBackground;
+    color: #F1F1F1;
+    background: #040404;
```

Comment:
Leave comment indicating this needs to match the value in interface_config pl.

Reply:
Wouldn't it make sense to have descriptive variables for these? Like `bodyBackground` etc?

Reply:
Considering it's only used here I don't see the benefit of adding a SCSS variable

Reply:
Ideally, we'd move everything to JSS so it would use the theme colors

Reply:
Makes sense.